services/ecosystems/src/util/appropriate_keys.js

```diff
@@ -7,20 +19,36 @@ module.exports = {
         return 'coverage_2018_S.tif';
       case 'T':
         return 'coverage_2018_T.tif';
+      case 'X':
       default:
         return '';
     }
   },
-  areaTypeKeys: (val) => {
-    switch (val) {
-      case 'basinZones':
-        return 'basin_zones';
-      case 'ea':
-        return 'environmental_authorities';
-      case 'basinSubzones':
-        return 'basin_subzones';
+  rasterCoverageSEKeys: (seType, coverageType) => {
+    switch (true) {
+      case seType === 'Bosque Seco Tropical' && coverageType === 'N':
+        return 'coverage_2018_N.tif';
+      case seType === 'Bosque Seco Tropical' && coverageType === 'S':
+        return 'coverage_2018_S.tif';
+      case seType === 'Bosque Seco Tropical' && coverageType === 'T':
+        return 'coverage_2018_T.tif';
+      case seType === 'Páramo' && coverageType === 'N':
+        return 'coverage_2018_N.tif';
+      case seType === 'Páramo' && coverageType === 'S':
+        return 'coverage_2018_S.tif';
+      case seType === 'Páramo' && coverageType === 'T':
+        return 'coverage_2018_T.tif';
+      case seType === 'Humedal' && coverageType === 'N':
+        return 'coverage_2018_N.tif';
+      case seType === 'Humedal' && coverageType === 'S':
+        return 'coverage_2018_S.tif';
+      case seType === 'Humedal' && coverageType === 'T':
+        return 'coverage_2018_T.tif';
+      case seType === 'Bosque Seco Tropical' && coverageType === 'X':
+      case seType === 'Páramo' && coverageType === 'X':
+      case seType === 'Humedal' && coverageType === '':
```

Comment:
I guess you're missing the `X` in this case?

Reply:
Yes, fixed!